fish-chips.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <a href="index.html">HOME</a>
+  <h1>FISH AND CHIPS RECIPE</h1>
+  <img src="images/fishchips.jpeg" alt="a picture of fish and chips">
+     <h2>INGREDIENTS</h2>
+    <ul>
+<li>7 tablespoons (55 grams) all-purpose flour, divided</li>
```

Comment:
is your eslint/prettier setup correctly? The formatting seems off here. Do you get auto formatting on saving of a file?